SETUP_NODE.md

```diff
@@ -0,0 +1,168 @@
+# Setup Hermez Node
+
+This tutorial will help you to setup an Hermez Node and run as sync mode.
+
+## Golang
+
+- [ ] [Install](https://golang.org/doc/install#install) Golang
+- [ ] [Create a](https://golang.org/doc/gopath_code#GOPATH) GOPATH environment
+- [ ] Set GOPATH/bin into PATH machine
+
+    ```command
+    export PATH=$PATH:$GOPATH/bin
+    ```
+
+## Go-Ethereum
+
+- [ ] Clone [GETH](https://github.com/ethereum/go-ethereum) project and do a `make all` command into project folder.
+  - Need have to gcc on PATH.
+
+    On Ubuntu/Debian you need install `build-essential`
+
+    ```command
+    sudo apt-get update
+    sudo apt-get install build-essential
+    ```
+
+- [ ] Add `build/bin` on PATH, after `make install` command inside of GETH
+folder.
+
+    ```command
+    export GETH_PATH=$GOPATH/src/github.com/ethereum/go-ethereum/
+    export PATH=$PATH:$GETH_PATH/build/bin
+    ```
+
+- [ ] Run the GETH
+
+    Enter in a empty folder (really) and you can run:
+
+    ```command
+    nohup geth --goerli --cache 4096 --http --http.addr INTERNAL_IP
+    --http.corsdomain *--http.vhosts* --http.api
+    "admin,eth,debug,miner,net,txpool,personal,web3" --ws --ws.addr INTERNAL_IP
+    --graphql --graphql.corsdomain *--graphql.vhosts* --vmdebug --metrics >
+    ~/temp/logs/geth-logs.log &
+    ```
+
+- [ ] Wait the geth command finished to move to next step.
+
+## PostgreSQL
+
+- [ ] [Install](https://www.postgresql.org/download/) PostgreSQL
+- [ ] Create a PostgreSQL User to project.
```

Comment:
```suggestion
- [ ] Create a PostgreSQL User called `hermez` to project.
```

Reply:
I think that this is not necessary, because on `cfg.builder.toml` the user can pass all information about PostgreSQL configs. 
But, what you think guys?

Reply:
It's necessary because some other stuff are linked to **hermez** database.

Reply:
Oh, got it!